steam/ext/csgo/enums.py

```diff
@@ -98,7 +98,25 @@ class ItemOrigin(IntEnum):
     LevelUpReward             = 24
     Max                       = 25
 
+class PerMapRankEnum:
```

Comment:
These should still be IntEnum subclasses, I meant change the names of the classes to remove Enum from the end

Reply:
Oh okay, my bad.